packages/json/package.json

```diff
@@ -1,17 +1,9 @@
 {
   "name": "@contentful/field-editor-json",
   "version": "3.3.6",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
+  "type": "module",
+  "main": "dist/esm/index.js",
   "types": "dist/types/index.d.ts",
-  "exports": {
```

Comment:
Do we need them or not?
I saw some post where it was mentioned that it was required for pure esm
other post are not mentioning them

Reply:
@andipaetzold going to tag you on this comment if that's okay? I don't think we need it, I've tested this in the web app and it works, but have also seen some articles recommending to have it in

Reply:
No, we don't need it